src/app/pages/login/login-page/login-page.component.scss

```diff
@@ -0,0 +1,144 @@
+$primary-color: #3cc355;
+$secondary-color: #798386;
+$text-color: #333;
```

Comment:
these defined properties are used elsewhere, can you have them defined globally and using them through out the application.